examples/proc_FID_nutation.py

```diff
@@ -44,31 +44,31 @@
     s /= prscr.zeroth_order_ph(s["t2":0])
     s.ft("t2")
     fl.image(s, ax=ax1)
-    d_raw = s.C
     ax1.set_title("Signal pathway / ph0")
     # }}}
     # {{{ Look at phase variation
-    phase_var_s = s.C
-    d_uncorrected = phase_var_s.C.real.integrate("t2")
+    d_uncorrected = s.C.real.integrate("t2")
+    # PR again, you need to rename these
```

Comment:
I have gone through multiple review cycles asking for this.  Even if it's on a different PR, you should have seen this when reading through the diff (are you proofing it before saying it's ready for review?)